modules/viz/include/opencv2/viz/types.hpp

```diff
@@ -132,6 +132,12 @@ namespace cv
             @param file File name (for now only PLY is supported)
              */
             static Mesh load(const String& file);
+
+            /** @brief Loads a mesh from a obj file.
+
+            @param file File name
+            */
+            static Mesh loadOBJ(const String& file);
```

Comment:
I would merge both load files and make decision on what to load based on extension.